bin/install

```diff
@@ -147,12 +160,19 @@ function get_url_from_version {
 
 function get_url_from_nightly {
   local os=$(get_os)
-  local arch_short=$(get_arch_short)
+  local arch_short=$(get_arch_short "$version")
   local extension=$(get_file_extension)
 
   case "$(get_os)" in
     "mac")
       local version_postfix="mac64"
```

Comment:
this line is covered by your changes below, can you remove it?

Reply:
So, I tried removing this line's change (7b3f5e2e199899fa225d4cf58c71285c6fb705f7), but without this change, I get the following output:

```
d-pai-mba:bin dilum$ asdf install julia nightly
/Users/dilum/.asdf/plugins/julia/bin/install: line 57: [: : integer expression expected
/Users/dilum/.asdf/plugins/julia/bin/install: line 58: [: : integer expression expected
/Users/dilum/.asdf/plugins/julia/bin/install: line 57: [: : integer expression expected
/Users/dilum/.asdf/plugins/julia/bin/install: line 58: [: : integer expression expected
/Users/dilum/.asdf/plugins/julia/bin/install: line 57: [: : integer expression expected
/Users/dilum/.asdf/plugins/julia/bin/install: line 58: [: : integer expression expected
```

When I re-add the change (7a0f71d5060a13b35a0146c75701094b6ed856b4), those error messages go away.

Reply:
Ah, I think I misunderstood your original comment. Were you talking about line 163, or line 168?

Reply:
Yes, line 168. The UI is kinda confusing when you click on the line number to add a comment. Thank you for the PR!